exchange.lua

```diff
@@ -985,8 +983,8 @@ function ex_methods.sell(self, p_name, ex_name, item_name, amount, rate)
 				return false, log2_err
 			end
 
-			remaining = 0
-			revenue = revenue + remaining * row.Rate
+			break
+			--revenue = revenue + remaining * row.Rate
```

Comment:
I guess that's right. I need to look again, maybe I had a bug in the original code.